casepropods/family_connect_registration/plugin.py

```diff
@@ -1,13 +1,36 @@
+from confmodel import fields
+from casepro.cases.models import Case
 from casepro.pods import Pod, PodConfig, PodPlugin
+import requests
 
 
 class RegistrationPodConfig(PodConfig):
-    pass
+    url = fields.ConfigText("URL to query for the registration data",
+                            required=True)
+    token = fields.ConfigText("Authentication token for registration endpoint",
+                              required=True)
 
 
 class RegistrationPod(Pod):
     def read_data(self, params):
-        pass
+        # Setup
+        url = self.config.url
+        token = self.config.token
+        session = requests.Session()
+        session.headers.update({'Authorization': "Token " + token})
+        session.headers.update({'Content-Type': "application/json"})
```

Comment:
I don't know if we need to create a session here since we're only doing a single request. We should be able to just do:
```python
r = requests.get(url, params=params, headers=headers)
```